tools/csdl-xml/test/cli.test.js

```diff
@@ -0,0 +1,84 @@
+const expect = require("chai").expect;
+
+const exec = require("child_process").exec;
+const fs = require("fs");
+const path = require("path");
+
+describe("CLI", () => {
+  it("Help", async () => {
+    const result = await cmd(["-h"]);
+    expect(result.code).to.equal(0);
+    expect(result.stdout).to.contain("Usage: csdl2xml");
+  });
+
+  it("Translate one file", async () => {
+    const outfile = "resources/jetsons.csdl.xml";
+    const basefile = "resources/jetsons.csdl.xml.base";
```

Comment:
Why `.xml.base` and not `.base.xml`? That would make it easier for an XML editor to recognize the file. And with VS Code extension `redhat.vscode-xml` installed and an XML catalog configured one even gets XML schema check for the "base" files for free.